app/controllers/devise_token_auth/concerns/set_user_by_token.rb

```diff
@@ -60,7 +60,9 @@ def set_user_by_token(mapping=nil)
       if devise_warden_user && devise_warden_user.tokens[@client_id].nil?
         @used_auth_by_token = false
         @resource = devise_warden_user
-        @resource.create_new_auth_token
+        # REVIEW: The following line _should_ be safe to remove;
+        #  the generated token does not get used anywhere.
+        # @resource.create_new_auth_token
```

Comment:
`update_auth_header` should use this, but if you say so, we can remove it

Reply:
Rather confusingly, there are _two_ methods with that name. 
the model concern:
https://github.com/lynndylanhurley/devise_token_auth/blob/38d27cfa80042c184c6f69e74c0789ca8ec0e407/app/models/devise_token_auth/concerns/user.rb#L199-L205
and the controller concern:
https://github.com/lynndylanhurley/devise_token_auth/blob/38d27cfa80042c184c6f69e74c0789ca8ec0e407/app/controllers/devise_token_auth/concerns/set_user_by_token.rb#L96-L154

I'll assume you mean the one in the controller concern (the same file as this change)...

Regardless, I'm not sure I understand what you mean. This is inside a conditional block for the use case of when `DeviseTokenAuth.enable_standard_devise_support = true` and the user has already authenticated with Devise  in the _session_ via Warden. There is no need for an auth token in that case, so why would we create one here?